liberty-maven-plugin/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -20,17 +20,47 @@
                     <lifecycle>
                         <id>default</id>
                         <phases>
+                            <pre-clean>
+                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:stop-server
+                            </pre-clean>
                             <process-resources>
-                                org.apache.maven.plugins:maven-resources-plugin:resources,
-                                net.wasdev.wlp.maven.plugins:liberty-maven-plugin:create-server
+                                org.apache.maven.plugins:maven-resources-plugin:3.0.2:resources
```

Comment:
Originally I didn't have a version, so it is not required, but when that warning popped on the recent build I realized there could be build variations without specifying a version.   The example in https://maven.apache.org/ref/3.3.9/maven-core/lifecycles.html show using versions.  In all cases, I chose the latest. If there is reason to remove it, I can.